lib/rules/no-multiple-empty-lines.js

```diff
@@ -113,8 +113,10 @@ module.exports = {
                 });
 
                 // add the notEmpty lines in there with a placeholder
-                notEmpty.forEach(function(x, i) {
-                    trimmedLines[i] = x;
+                notEmpty.forEach(function(nonEmptyLineNumber) {
+
+                    // FIXME: (not-an-aardvark) Use a better strategy after refactoring this rule
+                    trimmedLines[nonEmptyLineNumber] = "non-empty placeholder string";
```

Comment:
I disagree; in my opinion, overwriting the list with a bogus value is confusing for other people reading the code. Ideally, I think those lines would be marked explicitly (e.g. in a separate list/set), and then the separate set would be checked again when determining whether a line was empty.